xml/System.Security.Cryptography.X509Certificates/X509CertificateCollection.xml

```diff
@@ -780,9 +780,9 @@
         <Parameter Name="index" Type="System.Int32" Index="1" FrameworkAlternate="netcore-1.0;netcore-1.1;netstandard-1.3;netstandard-1.4;netstandard-1.6" />
       </Parameters>
       <Docs>
-        <param name="array">To be added.</param>
-        <param name="index">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="array">The one-dimensional array that is the destination of the elements copied from this collection. The array must have zero-based indexing.</param>
+        <param name="index">The zero-based index in <paramref name="array"/> at which copying begins.</param>
+        <summary>Copies the entire collection to a compatible one-dimensional array, starting at the specified index of the target array.</summary>
         <remarks>To be added.</remarks>
```

Comment:
We normally add a remark on explicit interface implementations like this. I'll give you a similar suggestion in all the EIIs so you can apply it:

```suggestion
         <remarks>
           <format type="text/markdown"><![CDATA[

## Remarks

This member is an explicit interface member implementation. It can be used only when the <xref:System.Security.Cryptography.X509Certificates.X509CertificateCollection> instance is cast to an <xref:System.Collections.ICollection> interface.  

           ]]></format>
         </remarks>
```

Reply:
That seems like something that would be better served in a template somewhere... but if that's what we're currently doing I guess I can hit add suggestion to batch :smile:.